src/js/components/Menu/stories/typescript/Children.tsx

```diff
@@ -10,9 +10,12 @@ const MenuWithChildren = (props) => (
       { label: 'Launch', onClick: () => {} },
       { label: 'Abort', onClick: () => {} },
     ]}
+    dropProps={{
+      a11yTitle: 'Test',
```

Comment:
if we are going to keep this story can we add something else other than `Test` or remove